### DISCUSSION TITLE
I can't access nextjs.org

### DISCUSSION THREAD

**Discussion by Markos-Th09** [vercel/next.js#40782] [Help] [answered]
Yesterday I sent a lot of requests to nextjs.org in a short amount of time and after that it seems I am not able to access it. Does vercel have any form of IP banning?

**Comment by leerob** [comment, marked as answer]
Yes, it's likely you were automatically blocked if you tried to automate sending a large number of requests to nextjs.org. 

Are you still blocked?

**Reply by Markos-Th09** [reply]
> Yes, it's likely you were automatically blocked if you tried to automate sending a large number of requests to nextjs.org.
> 
> Are you still blocked?

It seems temporary now I am not blocked. The weird thing is that it seemed like it was in the transport layer aka Connection Refused

**Comment by lnfel** [comment]
I left a tab with nextjs org website last night, now I can't access the thing. This is ridiculous.

**Reply by leerob** [reply]
Can you share a screenshot of what you're seeing? When you reload the page, do you see the same?

**Reply by lnfel** [reply]
I stopped visiting the site for several hours and was able to connect by afternoon. Then closed it after being done.

I tried to connect to the site the next day (today), same thing happened again:
<img width="649" alt="image" src="https://github.com/user-attachments/assets/d5357e45-359c-4803-a84f-2ac1de1347e7">

Pretty sure my ISP connection is fast enough since I can connect to other websites without any issue.

Also tried to ping and here is the result:
```
> ping nextjs.org
PING nextjs.org (76.76.21.98): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
Request timeout for icmp_seq 2
Request timeout for icmp_seq 3
^C
--- nextjs.org ping statistics ---
5 packets transmitted, 0 packets received, 100.0% packet loss
```

**Reply by lnfel** [reply]
Alright found some steps on how to resolve the issue albeit temporary, I am using a VPN now:
- https://github.com/vercel/vercel/issues/11084#issuecomment-2307777575
- https://vercel.community/t/resolving-ip-blocking-issues/171
